packages/cdktf/lib/terraform-iterator.ts

```diff
@@ -80,6 +81,26 @@ export abstract class TerraformIterator implements ITerraformIterator {
     return new MapTerraformIterator(map);
   }
 
+  /**
+   * Creates a new iterator from a resource that
+   * has been created with the `for_each` argument.
+   */
+  public static fromResources(
```

Comment:
This method is named in plural while the next is singular – I don't have any opinion but I'd want consistency 😄

Reply:
Should be plural on both indicating we are talking about lists of resources. Where lists means they were created through for each